static/app/views/performance/traces/tracesSearchBar.tsx

```diff
@@ -39,6 +56,9 @@ export function TracesSearchBar({
               'Search for traces containing a span matching these attributes'
             )}
             organization={organization}
+            metricAlert={false}
```

Comment:
This unfortunately has to be declared explicitly as false because of a tri-state boolean in SearchBar.